index.d.ts

```diff
@@ -1,13 +1,15 @@
 type Attributes = { [key: string]: any }
 
-type MessageResponse = { message_id: string, md5_of_body: string }
+export type MessageResponse = { message_id: string, md5_of_body: string }
+
+export type SendMessageOptions = undefined | { delay_seconds?: number, message_attribute?: Attributes }
```

Comment:
seems weird
```suggestion
export type SendMessageOptions = { delay_seconds?: number, message_attribute?: Attributes }
```

I think the optional argument in the `send_message` signature is sufficient?